src/js/utils/JobUtil.js

```diff
@@ -78,18 +78,27 @@ const JobUtil = {
       Object.assign(spec.run, { docker });
     }
 
+    // default
+    var scheduleId = "default";
+    var schedulePolicy = "ALLOW";
+    // preserve id and concurrencyPolicy
+    if (typeof spec.schedules != "undefined" && spec.schedules.length > 0) {
+      scheduleId = spec.schedules[0].id;
+      schedulePolicy = spec.schedules[0].concurrencyPolicy;
+    }
+
     // Reset schedules
     spec.schedules = [];
 
     // Only transfer schedule if checkbox is set, and create job with reasonable
     // defaults
     if (!schedule || schedule.runOnSchedule) {
       const {
-        id = "default",
+        id = scheduleId,
```

Comment:
The whole new behaviour here needs to be covered by unit tests on this file:

src/js/utils/__tests__/JobUtil-test.js